nav2_costmap_2d/include/nav2_costmap_2d/array_parser.h

```diff
@@ -28,13 +28,13 @@
  *
  * author: Dave Hershberger
  */
-#ifndef COSTMAP_2D_ARRAY_PARSER_H
-#define COSTMAP_2D_ARRAY_PARSER_H
+#ifndef nav2_costmap_2d_ARRAY_PARSER_H
```

Comment:
Looks like a global search and replace issue.

Reply:
Will be in next PR